homeassistant/components/overkiz/quality_scale.yaml

```diff
@@ -0,0 +1,68 @@
+rules:
```

Comment:
```py
@EVENT_HANDLERS.register(EventName.EXECUTION_REGISTERED)
async def on_execution_registered(
    coordinator: OverkizDataUpdateCoordinator, event: Event
) -> None:
    """Handle execution registered event."""
    if event.exec_id and event.exec_id not in coordinator.executions:
        coordinator.executions[event.exec_id] = {}

    if not coordinator.is_stateless:
        coordinator.update_interval = timedelta(seconds=1)
```
What is this for? It feels like you want to trigger an update of the listeners? Is that correct?

Reply:
We do poll for new events on the event queue at a predefined frequency (30 seconds for cloud, 5 seconds for local api) . This change will set this polling frequency to 1 second when a command has been executed via Home Assistant. This will make sure we do get the new state in HA as soon as it has been broadcasted. If the command execution is completed, it will move down to the original frequency again.